.github/unittest.yml

```diff
@@ -0,0 +1,29 @@
+name: Unittest xcube-clms
+
+on:
+  push:
+  release:
+    types: [published]
+
+jobs:
+  unittest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout xcube-clms
+        uses: actions/checkout@v4
+
+      - name: Set up MicroMamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+
+      - name: Run unit tests
+        shell: bash -l {0}
+        run: |
+          pytest --cov=xcube_clms --cov-report=xml
+
+      - name: Upload coverage reports to Codecov
```

Comment:
Let's see if this works out f the box. I think something needs to be set up on codecov webpage, then using the first time.